doc/src/content/xdocs/idl.xml

```diff
@@ -252,6 +252,14 @@ record Job {
   time_ms submitTime;
   timestamp_ms finishTime;
   decimal(9,2) finishRatio;
+}
+        </source>
+        <p>Logical types can also be specified via an annotation, which is useful for logical types for which a keyword does not exist:</p>
+        <source>
+record Job {
+  string jobid;
+  @logicalType("timestamp-micros")
+  long finishTime;
```

Comment:
Copied to the new website: https://github.com/martin-g/avro-website/commit/f685368ff8d211494c2ea84e059cc57d4bb3adf9

Reply:
Thanks @martin-g !